### PR TITLE
Adds styles for the widget modal

### PR DESCRIPTION
**Pull request by javierarce**
This PR adds styles for the modal widget and fixes a problem with the size of the avatar in the EditorMenu.

screen shot 2016-02-25 at 15 47 49
screen shot 2016-02-25 at 15 47 54
screen shot 2016-02-25 at 15 47 57

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @xavijam 
CC: @piensaenpixel

File: lib/assets/javascripts/cartodb3/editor-pane.tpl [hunk 1]
```diff
@@ -3,8 +3,8 @@
     <i class="CDB-IconFont CDB-IconFont-cartoFante u-actionTextColor"></i>
   </div>
   <ul class="EditorMenu-navigation js-menu"></ul>
-  <a href="<%- url %>" class="CDB-MenuEditor-avatar">
-    <img src="<%- avatar_url %>" class="CDB-MenuEditor-avatarImage">
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor-pane.tpl]: :joy:

**Comment by xavijam** [comment]
Go ahead neng!

File: lib/assets/javascripts/cartodb3/editor-pane.tpl (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/editor-pane.tpl]: Rookie mistake :P

**Pull request merged**